README.md

```diff
@@ -143,5 +143,26 @@ Welcome to our source speech repository about [mobx-state-tree](https://github.c
 
 ### Utils
 #### [#41](https://github.com/Dedetat/pimp-my-powerpuff/pull/41/files) - onPatch
+> Some debuging tools ?
```

Comment:
* move to Utils description (and add a verb)
* introduce quickly this topic (may be use the first point ?)

Reply:
debugging